app/src/main/res/layout-land/administrator_controls_download_permissions_view.xml

```diff
@@ -64,15 +66,16 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/topic_update_on_wifi_title_text_view" />
 
-      <androidx.appcompat.widget.SwitchCompat
+      <Switch
```

Comment:
If we use `Switch` the output is `ON, Switch` or `OFF, Switch`
If we use `SwitchCompat` the output is `checked, Switch, ON` or `not-checked, Switch, OFF`

Reply:
@BenHenning 

This might sound weird but I tried following things and result was completely unexpected.

1. Created a new project **My Application**. Defined `SwitchCompat` in `activity_main.xml` file.
```
    <androidx.appcompat.widget.SwitchCompat
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:checked="false"
        android:minWidth="48dp"
        android:minHeight="48dp"
        android:textOff="OFF"
        android:textOn="ON"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintLeft_toLeftOf="parent"
        app:layout_constraintRight_toRightOf="parent"
        app:layout_constraintTop_toTopOf="parent" />
```
Talkback gave following result for this: `OFF, Switch, Double-tap to toggle` -> Toggle -> `ON`

Then I updated the library from 'androidx.appcompat:appcompat:1.1.0' to `'androidx.appcompat:appcompat:1.2.0'`
Now the result was `not checked, Switch, OFF, Double-tap to toggle` -> Toggle -> `checked`

From this I thought maybe there is some library version issue so I downgraded library back to `androidx.appcompat:appcompat:1.1.0` but the result did not change and now it was fixed at checked/not-checked.

I was unable to reproduce this: `OFF, Switch, Double-tap to toggle` -> Toggle -> `ON`

So I again created a new project **Switch Test** but this time the result was  `not checked, Switch, OFF, Double-tap to toggle` -> Toggle -> `checked`

Now creating two seprarate new projects gave different results for first time which was unexpected.

From this, I think the issue should be with `Talkback` but I am not fully sure. Will investigate more on this.